local/README.md

```diff
@@ -0,0 +1,50 @@
+## Customizing mite docker-compose
+
+The docker-compose is setup to be able to run a separate docker container for each mite component. 
+
+
+### Running your own API server and performance test
+
+To run mite performance test against an app, the `apiserver` container in the docker-compose.yml can be exchanged with any docker image
+
+
+For exmaple, provide a dockerfile and run command
+
+```
+  apiserver:
+    build: 
+      context: .
+      dockerfile: Dockerfile.api
+    command: ["python", "manage.py", "runserver", "8000"]
+    ports:
+    - "8000:8000"
+
+```
+
+Modify the `demo_req` journey in local/demo.py to send a HTTP request to an endpoint from the custom apiserver
+
+```
+Note: The _API_URL listens to port 8000 of the apiserver container by default
+
+_API_URL = "http://localhost:8000" 
+
+@mite_http
+async def demo_req(ctx):
+    async with ctx.transaction("Custom request"):
+        await ctx.http.get(f"{_API_URL}/exampleendpoint")
+
+```
+
+Make sure to build a fresh mite image if there is code changes
```

Comment:
Would it make sense to use a docker mount to pull in the journeys, so that we don't need a new image if they change?  Something like https://docs.docker.com/storage/bind-mounts/#start-a-container-with-a-bind-mount (modified to fit into docker-compose).  This is not something that needs to be done for this PR, but a future suggestion of how to reduce the number of image rebuilds that are needed while testing code.

(Hard mode: the runner imports the journeys when it starts [or at latest when it runs the journey for the first time], and probably does not re-import them on change.  So the runners will need a restart when the journeys change.  To do that without a full image rebuild, the runner probably needs to watch the journey files with inotify or some such, and exit if it sees a change there...)

Reply:
That would be pretty nice, will take a look at those. Thanks!

Also, not sure what's going on with the CI checks 🤔